### ISSUE TITLE
How to generate yoshi_shapes_noStroke.svg and artistic mod?

### ISSUE THREAD

**Issue by lishunyao97** [wonga6/ACG_DepixelizingPixels#2]
I run your code and have yoshi_shapes.svg. Since there are strokes on it, I am sure that is not the final result. I cannot find any code about generating yoshi_shapes.svg and those artistic mod pictures. Could you please tell me how to generate them? Thanks a lot!

**Comment by wonga6** [comment]
On line 1255 in the main.cpp there is a place where the stroke color on the shapes is set. For "no stroke" (which I suppose might be a rather misleading name, sorry) I replaced the stroke color (currently black) with regionColor.

Also as a note I am no longer actively working on this project and I did not get all the way through the implementation of the paper because this was a final project for a class.
